### PR TITLE
Adjust default options

### PR DESCRIPTION
**Pull request by niklasravnsborg**
I missed these two options myself and added them in my fork but would like to merge them into the main repo:

- `finder._FXSortFoldersFirst`
- `NSGlobalDomain.AppleSpacesSwitchOnActivate`

Any feedback is warmly welcome. Thank you for your work!

### CODE REVIEW

**Review by Enzime — CHANGES_REQUESTED** [review]
Can you add these to the tests?

**Comment by Enzime** [comment]
Can you also add the finder option to the tests?

**Comment by niklasravnsborg** [comment]
@Enzime as i mentioned previously, currently there aren't any tests for the finder options so i wouldn't know where to put them...

**Comment by Enzime** [comment]
The same file is fine

**Comment by niklasravnsborg** [comment]
@Enzime Done, I've went ahead and added tests for all finder options.

File: tests/system-defaults-write.nix [hunk 1]
```diff
@@ -73,24 +85,25 @@
...
-  print("Did not find content from %s in %s" % (fixture, out), file=sys.stderr)
-  sys.exit(1)
-EOL
-  '') [
+  };
+  test = lib.strings.concatMapStringsSep "\n"
+    (x: ''
+      echo >&2 "checking defaults write in /${x}"
+      ${pkgs.python3}/bin/python3 <<EOL
+      import sys
```
Comment (Enzime) [inline tests/system-defaults-write.nix:98]: Was this indentation an intentional change?

Comment (niklasravnsborg) [reply tests/system-defaults-write.nix:98]: Yes, I reformatted the code with nixfmt to make it more consistent.

**Review by Enzime — APPROVED** [review]
LGTM

Thanks for the contribution

**Pull request merged**